allensdk/brain_observatory/behavior/metadata/behavior_ophys_metadata.py

```diff
@@ -59,8 +61,25 @@ def imaging_plane_group_count(self) -> int:
         return self._extractor.get_plane_group_count()
 
     @property
-    def indicator(self) -> str:
-        return 'GCAMP6f'
+    def indicator(self) -> Optional[str]:
+        """Parses indicator from reporter"""
+        reporter_substring_indicator_map = {
+            'GCaMP6f': 'GCaMP6f',
+            'GC6f': 'GCaMP6f',
+            'GCaMP6s': 'GCaMP6s'
```

Comment:
Is this a complete list? I see `GC6f` which maps to `GCaMP6f` but is it possible for there to be a `GC6s` (that would presumably map to `GcaMP6s`)?

Reply:
@njmei it has been discussed in the corresponding ticket. It is a complete list.